mobile-app/app/screens/AppNavigator/screens/Loans/components/BottomSheetVaultList.tsx

```diff
@@ -118,26 +120,39 @@ export const BottomSheetVaultList = ({
                   )}
                 />
               </View>
-              <View style={tailwind("flex items-end")}>
-                <CollateralizationRatio
-                  totalLoanAmount={new BigNumber(item.loanValue)}
-                  colRatio={new BigNumber(item.collateralRatio)}
-                  minColRatio={new BigNumber(item.loanScheme.minColRatio)}
-                />
-                <NumericFormat
-                  value={item.loanScheme.minColRatio}
-                  thousandSeparator
-                  displayType="text"
-                  renderText={(value) => (
-                    <ThemedTextV2
-                      style={tailwind("text-xs font-normal-v2 mt-1")}
-                      light={tailwind("text-mono-light-v2-700")}
-                      dark={tailwind("text-mono-dark-v2-700")}
-                    >
-                      {translate("", "min. {{value}}%", { value })}
-                    </ThemedTextV2>
-                  )}
+              {selectedVault && selectedVault.vaultId === item.vaultId && (
+                <ThemedIcon
+                  style={tailwind("h-full mt-0.5")}
+                  light={tailwind("text-green-v2")}
+                  dark={tailwind("text-green-v2")}
+                  iconType="MaterialIcons"
+                  name="check-circle"
+                  size={18}
+                  testID="playground_status_indicator"
```

Comment:
let's change to appropriate testid
```suggestion
                  testID="playground_status_indicator"
```